apps/meteor/server/email/IMAPInterceptor.ts

```diff
@@ -94,14 +103,22 @@ export class IMAPInterceptor extends EventEmitter {
 		});
 	}
 
+	reconnect(): void {
+		const loop = async (): Promise<void> => {
```

Comment:
The function is async but there's no async things happening inside

Reply:
oops, thanks for pointing it out, fixed.